modules/caddyhttp/proxyprotocol/listenerwrapper.go

```diff
@@ -38,32 +37,69 @@ type ListenerWrapper struct {
 	// Allow is an optional list of CIDR ranges to
 	// allow/require PROXY headers from.
 	Allow []string `json:"allow,omitempty"`
+	allow []*net.IPNet
```

Comment:
We should use `netip.Prefix` if possile.

Reply:
Done